### DISCUSSION TITLE
Scelta

### DISCUSSION THREAD

**Discussion by MrNico98** [daboynb/windows_scripts#7] [Ideas]
Su "Remove windows apps" sarebbe carino fare magari l'opzione, rimuovi tutto (tranne cattura, calcolatrice) e un menu a scelta

**Comment by daboynb** [comment]
La calcolatrice viene preservata, lo strumento di cattura viene rimosso. Questo serve a fare un debloat completo.
Se facessi un menu dovrei continuamente tenerlo aggiornato siccome le app cambiano.
I classici debloat rimuovono le app selezionate, il mio usa `Where-Object {$_.name -notmatch` per rimuovere tutto quello che non è fondamentale così da funzionare sempre, anche se le app cambiano.
